modules/op-viewing-and-downloading-sboms.adoc

```diff
@@ -0,0 +1,87 @@
+// This module is included in the following assemblies:
+// * create/working-with-pipelines-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-viewing-and-downloading-sboms_{context}"]
+= Viewing and Downloading SBOMs
+
+The PipelineRun details page provides an option to download or view SBOMs, enhancing transparency and control within your supply chain. SBOMs lists all the software libraries that a component uses. Those libraries can enable specific functionality or facilitate development.
+
+You can use an SBOM to better understand the composition of your software, identify vulnerabilities, and assess the potential impact of any security issues that may arise.
+
+image::sbom.png[Options to download or view SBOMs]
+
+.Prerequisites
+
+* You have link:https://docs.openshift.com/container-platform/4.14/web_console/web-console.html#web-console[logged in to the web console].
+
+* You have the appropriate link:https://docs.openshift.com/container-platform/4.14/authentication/using-rbac.html#default-roles_using-rbac[roles and permissions] in a project to create applications and other workloads in OpenShift Container Platform.
+
+.Procedure
+
+. In the any perspective, switch to the relevant project where you want a visual representation of SBOMs.
+
+. Add a task to create the SBOM link in the following format:
+
++
+*Example SBOM link task*
+
++
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: sbom-task # <.>
+  annotations:
+    task.output.location: results  # <.>
+    task.results.format: application/text
+    task.results.key: LINK_TO_SBOM  # <.>
```

Comment:
There is one more Optional annotation that you can mention here, presence of this annotation will make the link redirect to an external page.


```suggestion
    task.output.location: results  # <.>
    task.results.format: application/text
    task.results.key: LINK_TO_SBOM  # <.>
    task.results.type: external-link  # Optional: This will redirect to external page

```